exercises/chapter6/test/Main.purs

```diff
@@ -180,7 +181,8 @@ Note to reader: Delete this line to expand comment block -}
           Assert.equal (act m1 (act m2 a))
             $ act (m1 <> m2) a
         test "concrete" do
-          Assert.equal 15
+          Assert.assert "not one of [ 15, 125 ]"
+            $ flip elem [ 15, 125 ]
             $ act m1 a
```

Comment:
Downside of this change is that we lose the helpful "expected x, got y" test failure messages.

Reply:
A reader wouldn't know what to expect. But fixed for consistency anyway.